### PR TITLE
HW4

### PR DESCRIPTION
**Pull request by NastyaScherbinskaya**
Задание4: 
Создать два произвольных массива, подсчитать максимальное и минимальное значение их элементов. Вывести на экран.
Поменять в массивах максимальные и минимальные значения местами.

### CODE REVIEW

**Review by EgorAntonovich — CHANGES_REQUESTED** [review]
Create gitignore file. Your PR consist of a bunch of unnecessary files

File: TrainingHW4/TrainingHW4/Program.cs [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+        {
+            int Min = array[0];
+
+            for (int i = 0; i < array.Length; i++)
+            {
+                if (array[i] < Min)
+                {
+                    Min = array[i];
+                }
+
```
Comment (EgorAntonovich) [inline TrainingHW4/TrainingHW4/Program.cs:17]: Code style. Blank line.

File: TrainingHW4/TrainingHW4/Program.cs [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+            }
+
```
Comment (EgorAntonovich) [inline TrainingHW4/TrainingHW4/Program.cs:19]: Code style. Blank line.

File: TrainingHW4/TrainingHW4/Program.cs [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+
```
Comment (EgorAntonovich) [inline TrainingHW4/TrainingHW4/Program.cs:20]: Code style. Blank line.

File: TrainingHW4/TrainingHW4/Program.cs [hunk 4]
```diff
@@ -0,0 +1,120 @@
...
+            for (int i = 0; i < array.Length; i++)
+            {
+
+                if (array[i] > Max)
+                {
+                    Max = array[i];
+                }
+
```
Comment (EgorAntonovich) [inline TrainingHW4/TrainingHW4/Program.cs:36]: Code style. Blank line.

File: TrainingHW4/TrainingHW4/Program.cs [hunk 5]
```diff
@@ -0,0 +1,120 @@
...
+            }
+
```
Comment (EgorAntonovich) [inline TrainingHW4/TrainingHW4/Program.cs:38]: Code style. Blank line.

File: TrainingHW4/TrainingHW4/Program.cs [hunk 6]
```diff
@@ -0,0 +1,120 @@
...
+            return Max;
+
```
Comment (EgorAntonovich) [inline TrainingHW4/TrainingHW4/Program.cs:40]: Code style. Blank line.

File: TrainingHW4/TrainingHW4/Program.cs [hunk 7]
```diff
@@ -0,0 +1,120 @@
...
+        {
+
+            int[] array1 = { 1, 4, 5, 7, 5, -2, 56 };
+            Console.WriteLine($"Максимальное значение array1: {FindMaxValue(array1)} \n");
+            Console.WriteLine($"Минимальное значение array1: {FindMinValue(array1)} \n");
+            ChangeMax_and_Min(array1);
+
+            Console.WriteLine("Измененный array1:\n");
+            for (int i = 0; i < array1.Length; i++)
```
Comment (EgorAntonovich) [inline TrainingHW4/TrainingHW4/Program.cs:93]: Create function for output info. Code duplication.

File: TrainingHW4/TrainingHW4/Program.cs [hunk 8]
```diff
@@ -0,0 +1,120 @@
...
+        }
+
+
+
+        static void Main(string[] args)
+
+        {
+
+            int[] array1 = { 1, 4, 5, 7, 5, -2, 56 };
+            Console.WriteLine($"Максимальное значение array1: {FindMaxValue(array1)} \n");
```
Comment (EgorAntonovich) [inline TrainingHW4/TrainingHW4/Program.cs:88]: It will be in min/max function

File: TrainingHW4/TrainingHW4/Program.cs [hunk 9]
```diff
@@ -0,0 +1,120 @@
+﻿using System;
+
+namespace TrainingHW4
+{
+    class Program
+    {
+        static int FindMinValue(int[] array)
```
Comment (EgorAntonovich) [inline TrainingHW4/TrainingHW4/Program.cs:7]: Simplify `FindMinValue` `FindMaxValue` into one function.
I think you can output min/max values and return array of min/max values.

File: TrainingHW4/TrainingHW4/Program.cs [hunk 10]
```diff
@@ -0,0 +1,120 @@
...
+            return Max;
+
+        }
+
+
+        static void ChangeMax_and_Min(int[] array)
+        {
+            int Max = array[0];
+            int Min = array[0];
+
```
Comment (EgorAntonovich) [inline TrainingHW4/TrainingHW4/Program.cs:48]: Read this [article](https://docs.microsoft.com/en-us/dotnet/api/system.array.indexof?view=net-6.0) and tell me how we can simplify this function.

**Review by EgorAntonovich — CHANGES_REQUESTED** [review]
Fix it.

File: TrainingHW4/TrainingHW4/Program.cs (hunk 10 above)
Comment (NastyaScherbinskaya) [reply TrainingHW4/TrainingHW4/Program.cs:30]: we can use index to change min and max values

File: TrainingHW4/TrainingHW4/Program.cs [hunk 11]
```diff
@@ -0,0 +1,79 @@
...
+            }
+            Console.WriteLine($"\n Max value : {Max}, Min value : {Min}\n");
+        }
+
+        static void ChangeMaxMin(int[] array)
+        {
+            int Max = array[0];
+            int Min = array[0];
+
+            for (int i = 0; i < array.Length; i++)
```
Comment (EgorAntonovich) [inline TrainingHW4/TrainingHW4/Program.cs:31]: You have function for find min & max, why you didn't use it?

**Review by EgorAntonovich — CHANGES_REQUESTED** [review]
Fix issues

File: TrainingHW4/TrainingHW4/Program.cs (hunk 10 above)
Comment (NastyaScherbinskaya) [reply TrainingHW4/TrainingHW4/Program.cs:30]: i have a question))
i can change min and max value using index if i have only one max and min value
so I've chosen this way if there are more than one min/max values in array

**Review by EgorAntonovich — APPROVED** [review]

**Pull request merged**